streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -1403,17 +1404,58 @@ public void shouldNotLoopInfinitelyOnMissingMetadataAndShouldNotCreateRelatedTas
 
     @Test
     public void shouldUpdateClusterMetadataAndHostInfoOnAssignment() {
```

Comment:
This is unrelated to this change, I just happened to notice this test didn't seem to actually be testing anything at all